module-2/data-analysis-with-pandas/your-code/games_sales.ipynb

```diff
@@ -0,0 +1,1060 @@
+{
```

Comment:
- The first thing you should do when importing your data sets is to get an overview of your dataset characteristics. 
Run on your term pip install pandas-profiling and then import pandas-profiling on your notebook. 
In your notebook apply the following on your dataframe 
```Python 
pandas_profiling.ProfileReport(df)
```
This is the link to the web page https://pypi.org/project/pandas-profiling/1.4.0/
- Try to not put the year as integer in order to avoid including it in your correlation matrix. 
- As per your correlation matrix , the correlation between Global sales and critic score is quite low(0.245471), you could have take a look on other- sales variable and its relationship with other sales variables. 
- You could also have run some analysis on distribution of sales by genre, year of release, etc... Plotting your groupby/pivot_table.